ux.symfony.com/templates/main/demos.html.twig

```diff
@@ -24,9 +24,7 @@
 
     <div class="container-fluid container-xxl px-4 pt-4 px-md-5">
         <div style="display: grid; gap: 3rem; grid-template-columns: repeat(auto-fit, minmax(min(100%, 360px), 1fr));">
-            {% for demo in liveComponentDemos %}
-                <twig:Demo:DemoCard demo="{{ demo }}" />
-            {% endfor %}
+            <twig:DemosList />
```

Comment:
Really not sure here.. we cannot handle paginations, orders, ...

Reply:
Ok I will look at this